recourse_methods/mrmc_method.py

```diff
@@ -337,8 +346,9 @@ def cluster_data(
             cluster_centers=cluster_centers,
         )
 
-    def validate_cluster_assignments(
-        self, cluster_assignments: pd.DataFrame, k_directions: int
+    @staticmethod
+    def _validate_cluster_assignments(
+        cluster_assignments: pd.DataFrame, k_directions: int
     ) -> bool:
         """Raises an error if the cluster_assignments is valid.
```

Comment:
I think this should be "raises an error if the cluster_assignments are _invalid_".